### PR TITLE
Math operations op codes

### PR DESCRIPTION
**Pull request by vladNed**
Implemented the following opcodes:
- Add
- Sub
- Mult
- Div

### CODE REVIEW

File: src/vm/script.rs [hunk 1]
```diff
@@ -2460,6 +2460,114 @@ impl<'a> ScriptExecutor<'a> {
...
+                            (i_ptr, func_idx, op.clone(), exec_stack.as_slice()).into(),
+                        );
+                    }
+
+                    let mut last = exec_stack.pop().unwrap();
+                    *memory_size -= last.size();
+                    let mut second = exec_stack.pop().unwrap();
+                    *memory_size -= second.size();
+
+                    match second.sub(&mut last) {
```
Comment (octavonce) [inline src/vm/script.rs:2503]: This should be reversed i.e. the second item popped should be subtracted from the first.

File: src/vm/script.rs [hunk 2]
```diff
@@ -2460,6 +2460,114 @@ impl<'a> ScriptExecutor<'a> {
...
+                            (i_ptr, func_idx, op.clone(), exec_stack.as_slice()).into(),
+                        );
+                    }
+
+                    let mut last = exec_stack.pop().unwrap();
+                    *memory_size -= last.size();
+                    let mut second = exec_stack.pop().unwrap();
+                    *memory_size -= second.size();
+
+                    match second.div(&mut last) {
```
Comment (octavonce) [inline src/vm/script.rs:2557]: This should be reversed.

**Review by octavonce — CHANGES_REQUESTED** [review]

File: src/vm/script.rs (hunk 1 above)
Comment (vladNed) [reply src/vm/script.rs:2503]: just to clarify, the exec stack would be the inverse of the input stack. I guess if in the script we have `[4, 2, div]` we'll have in the exec stack as `[div, 2, 4]` so the second item pooped in the op code would be 4. 

Having it in reverse here would mean to divide 2 / 4. Am I correct ?

Comment (octavonce) [reply src/vm/script.rs:2503]: The input stack is spilled into the execution stack of the first frame in a `FIFO` manner. Assuming the first item in the following stack is the leftmost: `[10, 2]` the add instruction would pop both and divide `10` by `2` then push `5` to the execution stack: `[5]`.

**Review by octavonce — APPROVED** [review]

**Pull request merged**